CMakeLists.txt

```diff
@@ -39,11 +39,7 @@ EXEC_PROGRAM(
 )
 
 IF (SC_GLOBAL_PYTHON_WRAPPER)
-	IF (APPLE)
```

Comment:
Here's is the content of CMakeCache.txt when using your code to find Python libs (note: Python of OS X, no Homebrew Python installed)
//Path to a program.
PYTHON_CONFIG_EXECUTABLE:FILEPATH=/usr/bin/python-config

//Path to a program.
PYTHON_EXECUTABLE:FILEPATH=/usr/bin/python

//Path to where Python.h is found
PYTHON_INCLUDE_DIR:PATH=/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7

//Path to where libpython2.7.dylib is found
PYTHON_LIBRARY:PATH=/System/Library/Frameworks/Python.framework/Versions/2.7/lib/libpython2.7.dylib